Makefile

```diff
@@ -25,7 +25,7 @@ WHAT ?= ./pkg
 unit_test_args ?=  -r -keepGoing --randomizeAllSpecs --randomizeSuites --race --trace $(UNIT_TEST_ARGS)
 
 export KUBEVIRT_PROVIDER ?= k8s-1.17
-export KUBEVIRT_NUM_NODES ?= 2 # 1 master, 1 worker needed for e2e tests
+export KUBEVIRT_NUM_NODES ?= 3 # 2 master, 1 worker needed for e2e tests
```

Comment:
Not related.

Reply:
To test the leader election part of the poc we need two masters.